Site/LogMan.io/Release Notes/v25.28.md

```diff
@@ -0,0 +1,1008 @@
+# Release Notes v25.28
+
+## Patches
+
+### ASAB Maestro v25.28.01
```

Comment:
since these are release notes for LogMan.io product (Site/**LogMan.io**/Release Notes/v25.28.md) and since new maestro patch causes a new logman patch anyway (maestro vesrion is referenced in logman's `define` section), i think all the entries here should be for LogMan.io patches, not maestro.

or at least that's my understanding.

Reply:
or... now im not sure. logman's define section says `asab_maestro_library: v25.28.02`:

```yaml
define:
  type: rc/version
  product: LogMan.io
  version: v25.28.04
  asab_maestro_library: v25.28.02
```

@eliska-n should there really be a maestro **library tag** referenced in logman version file? or should it be the maestro version?